src/myr/reference.js

```diff
@@ -386,6 +393,24 @@ let animations = [
         description: <span>The setMagnitude function sets the magnitude attribute of the cursor in units or degrees. The default value is 1.</span>,
         example: "setMagnitude"
     },
+    {
+        name: "setAnimationTrigger",
+        parameters: [{ type: "string", name: "startEvent" }],
+        description: <span>The setAnimationTrigger function sets the startEvent attribute of the cursor in string. Lists are click, mouseEnter, mouseLeave, and none. The default is none</span>,
+        example: "setAnimationTrigger"
+    },
+    {
+        name: "setDelay",
+        parameters: [{ type: "number", name: "time" }],
```

Comment:
Please discussion with @elenaizotova the wording of this parameter. There might be a better word that time.